exercises/03_count_rune/03_count_rune.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+)
+
+func CountRune(s string, r rune) int {
+	var count int
+	for i := range []rune(s) {
```

Comment:
nit:
- string version of for-range defaults to runes:
- it's cleaner to use values

```go
for _, c := range s {
  if c == r {  count++ }
}
```